src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/Metacity.java

```diff
@@ -118,11 +118,7 @@ class Metacity implements SynthConstants {
             try {
                 INSTANCE = new Metacity(themeName);
             } catch (FileNotFoundException ex) {
```

Comment:
Is it possible that the FileNotFoundException was ignored intentionally? After this change, it will be traced.

Reply:
I didn't change how `FileNotFoundException` is handled. It's still ignored.